src/test/java/org/jboss/arquillian/extension/jacoco/test/unit/FilterComposerTestCase.java

```diff
@@ -68,36 +68,36 @@ public void should_exclude_asset_not_matching_include_pattern() throws Exception
    }
 
    @Test
-   public void should_include_asset_not_matching_exclude_pattern() throws Exception
+   public void should_include_asset_matching_one_of_include_patterns() throws Exception
    {
       // given
-      final Filter<ArchivePath> filter = createComposedFilter(null, "org.arquillian.extension.jacoco.client.*");
+      final Filter<ArchivePath> filter = createComposedFilter("org.arquillian.extension.jacoco.client.*, org.arquillian.extension.integration.*", null);
 
       // when
-      final boolean include = filter.include(ArchivePaths.create("org/arquillian/extension/jacoco/container/CoverageDataCommand.class"));
+      final boolean include = filter.include(ArchivePaths.create("org/arquillian/extension/jacoco/client/ManifestAsset.class"));
```

Comment:
No, the class is still in the same package. I re-ordered a few test cases and now the diff shows something weird.

But what do you mean by refactoring friendly ?